Makefile

```diff
@@ -27,7 +27,7 @@ DOCKER_IMAGE_DIRS := $(patsubst %/Dockerfile,%,$(DOCKERFILES))
 BUILD_IN_CONTAINER ?= true
 
 # ensure you run `make drone` after changing this
-BUILD_IMAGE_VERSION := 0.20.4
+BUILD_IMAGE_VERSION := publish-compiled-mixin-a795b93-WIP
```

Comment:
@pracucci I don't if  WIP suffix is intentional? I think WIP suffix comes from the fact, you may have some uncommitted changes while building image via `make loki-build-image`.

We can get rid of it by removing all git modified files in local workspace.

Can we keep some static tag instead? (e.g: `0.20.5`)

Reply:
Yeah, see PR description:

> Draft PR to get a feedback. Then I will open a dedicated PR to properly update the build image.

Given you're approving this PR, I'm now going to open a PR to publish 0.20.5 (following the 2 steps process in your instructions).

Reply:
See: https://github.com/grafana/loki/pull/6255